### PR TITLE
Add myself to contributors

### PR DESCRIPTION
**Pull request by nicogodet**
## Description

It's time after couple of merged (and unmerged) PR !

### CODE REVIEW

**Comment by agiudiceandrea** [comment]
Consider also updating https://github.com/qgis/QGIS/blob/master/doc/CONTRIBUTORS

File: resources/data/contributors.json [hunk 1]
```diff
@@ -1145,6 +1145,23 @@
           -26.62229
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "Name": "Nicolas Godet",
+        "Committer": "No",
+        "First Commit Message": "Fix qgis#41977",
```
Comment (nyalldawson) [inline resources/data/contributors.json:1154]: I realise it's not the exact commit message, but I'd rather see something more descriptive here like "Fix QgsCellStatisticsAlgorithm::name() should not return a translatable string".

**Comment by nyalldawson** [comment]
Nice @nicogodet , certainly well earned!

File: resources/data/contributors.json (hunk 1 above)
Comment (nicogodet) [inline resources/data/contributors.json:1154]:
```suggestion
        "First Commit Message": "Fix QgsCellStatisticsAlgorithm::name() should not return a translatable string",
```

**Comment by nicogodet** [comment]
> Nice @nicogodet , certainly well earned!

Thank you!

Let's squash this

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**